### PR TITLE
Fixes wrong link in the PULL_REQUEST_TEMPLATE.md file

### LINKED ISSUE #14: Fix wrong link in the PULL_REQUEST_TEMPLATE.md file
**Actual Behaviour**

The `Pull Requests` link in the 2nd checklist links to `../../pulls`. The URL of a Pull Request is of the format `https://github.com/NJACKWinterOfCode/HackerRank-Test-Case-Generator/pull/<pull-reuest-number>`. So, ultimately the `Pull Requests` links to https://github.com/NJACKWinterOfCode/pulls which is wrong.
This can be corrected each time when opening a Pull Request but it is better to correct it once and for all in the PULL_REQUEST_TEMPLATE.md file

**Expected Behaviour**

The `Pull Requests` should link to https://github.com/NJACKWinterOfCode/HackerRank-Test-Case-Generator/pulls 

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by agarwalrounak**
### Checklist:

* [x] Have you followed the guidelines in our Contributing document?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?
* [x] Have you successfully ran tests with your changes locally?
* [x] Have you added an explanation of what your changes do and why you'd like us to include them?

### Issues Solved

Fixes #14

### CODE REVIEW

**Comment by agarwalrounak** [comment]
@aashutoshrathi please review! It's been two weeks!

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ### Checklist:
 
 * [ ] Have you followed the guidelines in our Contributing document?
-* [ ] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?
+* [ ] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?
```
Comment (aashutoshrathi) [inline .github/PULL_REQUEST_TEMPLATE.md:4]: Still not working :confused:

**Review by aashutoshrathi — CHANGES_REQUESTED** [review]
Doesn't work for me.

**Comment by agarwalrounak** [comment]
Consider the PRS #15 and #13. In  these PRs, try clicking on the `Pull Requests` link in the 2nd option of the checklist. You wiil get a page not found error. Now, try clicking on the `Pull Requests` link in PR #12. You will be redirected to the correct page. This is because I edited the link URL from ../../pulls to ../pulls in this PR. 
Thus, PR #15 aims to correct this error once and for all in the PULL_REQUEST_TEMPLATE.md file.

**Comment by aashutoshrathi** [comment]
I checked on this PR itself, since the path is relative this works only through PR pages.

**Review by aashutoshrathi — APPROVED** [review]

**Pull request merged**